src/simplewallet/simplewallet.cpp

```diff
@@ -3022,7 +3216,13 @@ simple_wallet::simple_wallet()
                                   "device-name <device_name[:device_spec]>\n "
                                   "  Device name for hardware wallet.\n "
                                   "export-format <\"binary\"|\"ascii\">\n "
-                                  "  Save all exported files as binary (cannot be copied and pasted) or ascii (can be).\n "));
+                                  "  Save all exported files as binary (cannot be copied and pasted) or ascii (can be).\n "
+                                  "persistent-client-id <1|0>\n "
+                                  "  Whether to keep using the same client id for RPC payment over wallet restarts.\n"
+                                  "auto-mine-for-rpc-payment-threshold <float>\n "
+                                  "  Whether to automatically start mining for RPC payment if the daemon requires it.\n"
+                                  "credits-target <unsigned int>\n"
+                                  "  The RPC payment credits balance to target (0 for default " BOOST_STRINGIZE(CREDITS_TARGET) ")."));
```

Comment:
Is it OK to have a macro call within `tr(...)`?

Reply:
Good point, it'll end up within the translatable string, I'll change.